insights/parsers/installed_product_ids.py

```diff
@@ -13,7 +13,7 @@
 from insights.specs import Specs
 from .. import parser, CommandParser
 
-add_filter(Specs.subscription_manager_installed_product_ids, 'ID:')
+add_filter(Specs.subscription_manager_installed_product_ids, ['ID:', 'Product Certificate', 'Path'])
```

Comment:
IIUC, the added `'Path'` is used to filter the lines like `Path: /etc/pki/product-default/69.pem` .  If this is the case, what about adjusting the filter string to `'Path: '` to gain more accuracy?

Reply:
sure, I'll update it.